www/pages/reuse_last_values.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+permalink: /howto/reuse_entered_values
+---
+
+<!--- IMPORTANT: This permlink is referenced from InputApp -->
```

Comment:
I don't think it is referenced directly from input

Reply:
True, it is not. That was a copy&paste. However, shall we add it somewhere to the app?

Reply:
In theory we could have a "learn more" link in the Settings near that option... but from the time being I am happy to leave it as is and just remove the misleading comment

Reply:
Sure, done.